modules/core/shared/src/main/scala/gem/math/Wavelength.scala

```diff
@@ -54,4 +31,11 @@ object Wavelength {
   implicit val WavelengthOrd: Order[Wavelength] =
     Order.by(_.toAngstroms)
 
+  /**
+   * Prism from Int into Wavelength and back.
+   * @group Optics
+   */
+  def fromAngstroms: Prism[Int, Wavelength] =
```

Comment:
Is the `from...` prefix in the name conventional for monocle prisms? It sounds odd to me since they are bidirectional.

Reply:
My reasoning was that the prism really should be on `Int` because you're "narrowing" it to `Wavelength`, but since it's on `Wavelength` I added `from-` to indicate that the sense is reversed. I'm not super happy with it. Open to suggestions.

Reply:
I mean, we *could* use syntax to add it to `Int`, so you would say `Int.angstroms.getOption(42)`, it just seems like it might be hard to discover. I'm new to this so I don't really have a feel for it.

Reply:
With Haskell `lens` it would be `_Ansgtroms`, but I've never seen that in monocle. There must be some stock prisms coming with monocle, how are they named them?

Reply:
I don't know if there's a convention. We could do `_optic` if we wanted to.

Reply:
In any case I'm going to merge this and we can go back and rename things if we come up with a scheme we like better.

Reply:
From the monocle examples I've seem it'd be named either just `angstroms` or `angstromsToInt` which isn't great. In any case I guess the `from-` is better than any other naming scheme I could think of.